jeverd/lecture-experience
# Upload PDF Document to the Board

**Purpose**
The purpose of this ticket is to allow the host to upload their PDF document to their whiteboard.
**How to test**
1. Create a new lecture
2. Click the add image button on the top left toolbar
3. Select a PDF file
4. Verify that all of the pages of the PDF file have been added on separate boards. Once the pages have been uploaded, you should be redirected to the board with the first page of the pdf.
**Screenshots & Other Images/Videos**
![Screen Shot 2020-09-09 at 9 41 47 AM](https://user-images.githubusercontent.com/31896750/92606207-b125c680-f280-11ea-81ff-08431d68e7c5.png)
